kinks/level_4/introduction_to_programming_i/README.md

```diff
@@ -123,15 +129,50 @@ Note that the `distance`/`km`/`miles_travelled` property is increasing towards t
 
 Once implemented correctly, remember the logic, as you will likely need to create similar functions things in the stages ahead.
 
-#### 702, stage 4
+#### 702, Stage 4
 _"My suspect's car didn't have enough space to pull out in front of me before speeding off, so the chase wouldn't work as expected. I deliberately got myself suspended so I could grab another case, and it worked fine."_ (reported by [@dannycallaghan](https://github.com/dannycallaghan))
 
-#### 802, stage 2
+#### 801, All stages
+More spotlight stages, which may cause Firefox users to not see the spotlights. 
+To get it to work, comment out `blendMode(DARKEST)` during your work and to uncomment again before submission. Or don't.
+
+Tips:
+- The 1st seat in the 1st row will be `seats[0][0]` 
+- Rows and seats relate to `seats[row][seat]` and not vice versa
+- You don't need to factor in reverse counting of rows such that front to back make row numbers increase like in a real theatre. For this case the first row is at the back of the theatre (i.e. the top of the screen)  
+
+#### 801, Stage 3-4
+- Many students waste time here trying to spot patterns in the audience members based on their appearance. The information about the seat numbers of gang/perpetrators is already defined in an array. It appears that any information given about appearance (hats, retro glasses, etc.) is merely to confirm that you have used the array to traverse the seats in the correct manner and not to actually identify gang members.
+
+#### 801, Stage 4
+Here the challenge is to traverse a two-dimensional array using a normal (one-dimensional) array. There are different ways to approach this, but these work:
+
+- create additional variables and use a nested-for with conditionals, or
+- convert the 1D array to a 2D array and compare against the 2D array
+
+It appears that the grader is looking for two simple nested for-loops in the solution, so more elegant ways involving math to convert your 1D iterator into two 2D co-ordinates does not work, even when all gang members are correctly identified. This, for example, was not accepted by the grader: 
+`for (var i = 0; i < suspect.length; i++)
+operaFolks[floor(i/10)][i%10].recognised = (suspect[i]);`
```

Comment:
This code will not render as expected, it will stay on one line:
`for (var i = 0; i < suspect.length; i++)
operaFolks[floor(i/10)][i%10].recognised = (suspect[i]);`

A block code is needed for this purpose (starts with 4 spaces, surrounded with empty lines):

    for (var i = 0; i < suspect.length; i++)
    operaFolks[floor(i/10)][i%10].recognised = (suspect[i]);

Also, even though it does not produce a syntax error as such, it is best for clarity to keep curly braces around expressions where they are usually expected.

Reply:
Code block added.